docs/installation/tools-for-developers/index.md

```diff
@@ -1,5 +1,51 @@
 # Installation of tools for developers
 
-!!! warning
+## ROS 2 settings
 
-    Under Construction
+### Colorizing logger output
+
+By default, ROS 2 logger doesn't colorize the output.
+To colorize it, write the following in your `.bashrc`:
+
+```bash
+export RCUTILS_COLORIZED_OUTPUT=1
+```
+
+### Customizing the format of logger output
+
+By default, ROS 2 logger doesn't output detailed information such as file name, function name, or line number.
+To customize it, write the following in your `.bashrc`:
+
+```bash
+export RCUTILS_CONSOLE_OUTPUT_FORMAT="[{severity} {time}] [{name}]: {message} ({function_name}() at {file_name}:{line_number})"
+```
+
+See [here](https://docs.ros.org/en/rolling/Tutorials/Logging-and-logger-configuration.html#console-output-formatting) for more options.
+
+### Enabling localhost-only communication
+
+By default, ROS 2 communicates using multi-cast, which may unnecessarily increase the network traffic.
+To avoid it, write the following in your `.bashrc`:
+
+```bash
+export ROS_LOCALHOST_ONLY=1
+```
+
+### Setting up `ROS_DOMAIN_ID`
+
+ROS 2 nodes on the same domain can freely discover and send messages to each
+other, while ROS 2 nodes on different domains cannot. All ROS 2 nodes use
+domain ID 0 by default. To avoid interference between different groups of
+computers running ROS 2 on the same network, a different domain ID should
+be set for each group.
+
+See [here](https://docs.ros.org/en/foxy/Concepts/About-Domain-ID.html#the-ros-domain-id)
+for more information.
```

Comment:
I'd like to discuss the style of Markdown files.
Since Markdown will automatically set the layout, I think we don't have to adjust it manually.

The disadvantages of writing like this are:
- The positions of period marks become unclear.
- People have to adjust the layout when they edit it.

If it's `DISCLAIMER` like [this](https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto/-/blob/master/DISCLAIMER.md), people might read it in the raw format, it's fine to adjust the layout.
But I believe we don't need it for Markdown files.

What do you think about it? @xmfcx

Reply:
I've simplified it, how about it? dd12560c2c44e8114cf50ce3ef1b08e89cddd858

Reply:
@kenji-miyake why not use something like https://github.com/prettier/prettier for formatting markdown files automatically?

Reply:
@xmfcx I already use it, but I guess this isn't a supported use case.

Reply:
FYI: It's automatically checked using `pre-commit`.

https://github.com/autowarefoundation/autoware-documentation/runs/5513786672?check_suite_focus=true#step:3:112
https://github.com/autowarefoundation/autoware-documentation/blob/41eeea7b6be1ffbd92fed2e4bc892437a7db88d3/.pre-commit-config.yaml#L27-L30

Reply:
I get it, I'm ok with your choice on this.